tools/bazar/actions/BazarListeAction.php

```diff
@@ -179,84 +195,50 @@ public function run()
                 && (!isset($this->arguments['calledBy']) || $this->arguments['calledBy'] !== 'CalendrierAction')) {
             return $this->callAction('calendrier', $this->arguments);
         }
-
-        $entryManager = $this->getService(EntryManager::class);
-        $formManager = $this->getService(FormManager::class);
-        $externalWikiService = $this->getService(ExternalBazarService::class);
-
-        if (!isset($GLOBALS['_BAZAR_']['nbbazarliste'])) {
-            $GLOBALS['_BAZAR_']['nbbazarliste'] = 0;
-        }
-        ++$GLOBALS['_BAZAR_']['nbbazarliste'];
-
-        // TODO put in all bazar templates
-        $this->wiki->AddJavascriptFile('tools/bazar/libs/bazar.js');
-
-        // External mode activated ?
-        if ($this->arguments['externalModeActivated']) {
-            $forms = $externalWikiService->getFormsForBazarListe($this->arguments['externalIds'], $this->arguments['refresh']);
-            $entries = $externalWikiService->getEntries([
-                'forms' => $forms,
-                'refresh' => $this->arguments['refresh'],
-                'queries' => $this->arguments['query']
+        
+        $bazarListService = $this->getService(BazarListService::class);
+        $bazarListService->setArguments($this->arguments);
+        $forms = $bazarListService->getForms();
+
+        if ($this->arguments['dynamic']) {
+            return $this->render("@bazar/entries/index-dynamic-templates/{$this->arguments['template']}.twig", [
```

Comment:
@seballot So if I understand, you propose to use a new folder for templates for bazarliste, whereas we have already the folder /tools/bazar/templates for twig templates. Is-it right ?

Reply:
`tools/bazar/templates` c'est pour tous les tmeplates de bazar, donc ça me paraitrait bizarre de mettre la (potentiellement longue) liste de template de bazarListAction direct dans ce dossier non? c'est pourquoi j'ai fait un sous dossier : `tools/bazar/templates/entries/index-dynamic-templates`. T'en penses quoi?

Reply:
C'est peut etre juste le nom du dossier qui est trop long, genre le mot `templates` ne sert pas à grand chose, et `index-dynamic` n'est pas super parlant. Peut etre aller vers `tools/bazar/templates/entries/dynamic` ou `tools/bazar/templates/entries/client-side` ?

Reply:
Effectivement @mrflos je suis d'accord avec ta proposition de `tools/bazar/templates/entries/dynamic`